### PR TITLE
Reworking how facilities_api handles PPMS pagination

### PR DESCRIPTION
**Pull request by darkbushido**
## Description of change

PPMS gave us some new endpoints that support pagination
this pr removes all of the pagination hacks we have used in the past.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#32812

### CODE REVIEW

File: config/initializers/statsd.rb [hunk 1]
```diff
@@ -203,8 +203,17 @@
...
 
       if params['radius']
+        count = case payload.body
+                when Hash
+                  payload.dig(:body, :value) || 0
+                when Array
+                  payload.body&.count || 0
+                else
+                  0
+                end
```
Comment (ericboehs) [inline config/initializers/statsd.rb:213]:
```suggestion
        count =
          case payload.body
          when Hash
            payload.body[:value]
          when Array
            payload.body.count
          end
```

File: config/initializers/statsd.rb [hunk 2]
```diff
@@ -203,8 +203,17 @@
...
+
         tags << "facilities.ppms.radius:#{params['radius']}"
-        tags << "facilities.ppms.results:#{payload[:body].is_a?(Array) ? payload[:body]&.count : 0}"
+        tags << "facilities.ppms.results:#{count}"
```
Comment (ericboehs) [inline config/initializers/statsd.rb:216]:
```suggestion
        tags << "facilities.ppms.results:#{count || 0}"
```

File: config/initializers/statsd.rb (hunk 1 above)
Comment (ericboehs) [reply config/initializers/statsd.rb:213]: What do you think of moving the multiple `|| 0` down to where it's accessed?

**Review by ericboehs — APPROVED** [review]
Small refactor suggestion.

File: config/initializers/statsd.rb (hunk 1 above)
Comment (darkbushido) [reply config/initializers/statsd.rb]: I'm worried about the & case

File: config/initializers/statsd.rb (hunk 2 above)
Comment (darkbushido) [reply config/initializers/statsd.rb:251]: missed this, I'll give it a shot

**Review by rileyanderson — APPROVED** [review]

**Pull request merged**